### PR TITLE
fix: jobs page

### PR DESCRIPTION
**Pull request by vinckr**
Since there are a number of issues on the jobs page, I took the liberty of bundling them all in this PR, will close the duplicate PRs. 

changes in this PR: 
- adds the sentence about ory in general to the shared "about" part, removes it from the marketing jobs and the shared "dev"/"sales" parts
- renames the files to match the path
- renames shared imports dev/sales to be more descriptive
- changes the growth filetype to .md so it is listed and its path for better readability
- adds the changes from @marneysof from the PRs:
https://github.com/ory/web/pull/478/
https://github.com/ory/web/pull/479/
https://github.com/ory/web/pull/480/
https://github.com/ory/web/pull/481/
https://github.com/ory/web/pull/482/
https://github.com/ory/web/pull/483/
https://github.com/ory/web/pull/484/

Please review this thoroughly and with attention to detail in the preview @marneysof @BlueAcidFrog 🙏 
Make sure _all_ the jobs are listed on the /jobs page (growth marketer was missing) and check the descriptions & frontmatter (title,position,location etc.) for every job.

### CODE REVIEW

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 1]
```diff
@@ -6,24 +6,20 @@ metaTitle: 'Open Marketing Careers at Ory'
...
 
-Ory is the largest open-source authentication and authorization (login,
-registration, MFA, permissions, OAuth2) ecosystem and community in the world. We
-develop open-source software and use it to build a scalable, easy-to-use, and
-secure cloud platform. You will provide organizational support to Ory leadership
-to ensure smooth operations and communications. Together with the CEO, you will
-provide a core leadership function that collaborates with UX design,
-engineering, finance, marketing, and data analytics.
+As Marketing Campaign Manager, you will devise and execute marketing campaigns
+to expand Ory’s reach among prospects and customers across multiple channels.
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:22]:
```suggestion
As Marketing Campaign Manager you will strategize, execute, and optimize data-driven demand generation campaigns on an international level to expand Ory’s reach among prospects and customers across multiple channels.
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 2]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
 
 ### What You'll Do
 
-- Conceptualize and execute marketing campaigns to expand Ory’s reach among prospects and customers across multiple channels
+- Conceptualize and execute marketing campaigns to expand Ory’s reach among
+  prospects and customers across multiple channels
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:33]:
```suggestion
- Conceptualize, execute and optimize data-driven demand generation campaign to expand Ory’s reach among
  prospects and customers across multiple channels.
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 3]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
...
 - Create and manage a Digital Marketing campaign calendar
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:34]:
```suggestion
-  Develop and deploy integrated marketing campaigns aligning with internal teams, developing audience segments, building up an E-Mail base, and planing top-funnel virtual events.
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 4]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
...
 - Assist team workflow and communicate across different teams
 - Collaborate with external Stakeholder when necessary
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:36]:
```suggestion
- Collaborate with internal and external Stakeholders to ensure proper messaging, timing, and execution of multi-channel campaigns
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 5]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
...
-- Research competitor strategies and industry standards in brand and performance campaigns
-- Benchmark innovative and flexible content creation and campaign management workflows
-- Make creative, engaging, and impactful marketing content in collaboration with team members
+- Research competitor strategies and industry standards in brand and performance
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:37]:
```suggestion
- Apply an experimentation mindset to marketing campaigns in order to drive, test, learn and optimize. Continuously conduct A/B testing across all channels
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 6]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
...
+  campaigns
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:38]:
```suggestion
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 7]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
...
+- Establish innovative and flexible content creation and campaign management
+  workflows
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:40]:
```suggestion
- Establish innovative and flexible content for our audience of developers and IT leaders and build prescriptive persona-based buyer journeys as the foundation for campaign strategy
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 8]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
...
+- Make creative, engaging, and impactful marketing content in collaboration with
+  team members
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:42]:
```suggestion
- Make creative, engaging, and impactful marketing content in collaboration with
  team members for our audience of developers and IT leaders
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 9]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
...
 - Grow and shape Ory’s presence as global brand with open source roots
 - Explore new channels for reaching relevant audiences for Ory Cloud
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:41]:
```suggestion
- Explore new channels for reaching relevant audiences for Ory Cloud
```
```suggestion
Provide ongoing reporting and campaign insights
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 10]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
...
 
-
 ### We’re looking for someone who has
 
 - 5+ years experience in digital and performance marketing
-- Strong writing skills and ability to explain technical concepts to a wide audience with varying levels of technical expertise
-- Experience in creative development and production and collaboration with external stakeholders 
-- Advanced understanding of industry standards and emerging trends in digital marketing
+- Strong writing skills and ability to explain technical concepts to a wide
+  audience with varying levels of technical expertise
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:47]:
```suggestion
  audience with varying levels of technical expertise
```
```suggestion
  - Experience measure tactic and channel impacts
```
```suggestion
  - Experience working in a B2B software marketing team
```
```suggestion
Experience with A/B testing, experiment design, campaign analysis (attribution reporting is a plus)
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 11]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
...
+- Experience in creative development and production and collaboration with
+  external stakeholders
+- Advanced understanding of industry standards and emerging trends in digital
+  marketing
 - Ability to self-manage and collaborate across teams as well as reporting
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:52]:
```suggestion
- Strong project management, organizational, and communication skills and ability to self-manage and collaborate across teams
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 12]
```diff
@@ -33,27 +29,33 @@ engineering, finance, marketing, and data analytics.
...
-- Generating marketing insights and willingness to try out creative new ideas and iterate on them 
-- Fluency in written and oral English
-
+- Generating marketing insights and willingness to try out creative new ideas
+  and iterate on them
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketing-campaign-manager.mdx:54]:
```suggestion
- Willingness and flexibility to work both independently as well as part of a team
```

File: src/markdown/jobs/marketing-campaign-manager.mdx (hunk 10 above)
Comment (vinckr) [reply src/markdown/jobs/marketing-campaign-manager.mdx:47]: ????

Comment (vinckr) [reply src/markdown/jobs/marketing-campaign-manager.mdx:47]: which one is it? 
merging the last one.

File: src/markdown/jobs/growth-marketing-manager.mdx [hunk 13]
```diff
@@ -33,28 +31,41 @@ engineering, finance, marketing, and data analytics.
...
 
-- Build programs and digital campaigns to help people understand, try, and adopt Ory products. 
-- Optimize through experimentation. Create A/B tests to iterate on content to increase engagement and product adoption. 
-- Distribute Ory Corp core messaging across digital channels. Create and expand digital channels to engage prospects and customers.
-- Build and adjust marketing operations infrastructure that incorporates performance measurements and analytics. 
-- Build campaigns (incl. retargeting) and perform segmentation analysis for personalized conversion journeys across different groups of users.
-- Collaborate with team members (and perhaps an external agency) to execute on SEO priorities to increase organic traffic to Ory Corp and associated websites.
-- Collaborate with other departments (HR, Sales, Engineering) to maximize yield from shared channels and programs.
+- Build programs and digital campaigns to help people understand, try, and adopt
+  Ory products.
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/growth-marketing-manager.mdx:35]:
```suggestion
- Develop and execute integrated B2B demand generation programs to help people understand, try, and adopt
  Ory products and achieve revenue and pipeline goals
```

**Review by BlueAcidFrog — DISMISSED** [review]

**Review by BlueAcidFrog — APPROVED** [review]

**Review by jfcurran — APPROVED** [review]

**Pull request merged**